packages/manager/.size-limit.cjs

```diff
@@ -0,0 +1,26 @@
+const pkg = require("./package.json");
+
+module.exports = [
```

Comment:
Size Limit tracks bundled + minified package size difference between PRs. It helps ensure packages do not get too large, which is especially important for packages used in the browser.

See the Size Limit repository for more info: https://github.com/ai/size-limit

You can see an example of the output here: https://github.com/prismicio/prismic-next/pull/29#issuecomment-1136687119

The SM repo is not currently configured to use Size Limit, but it can still be run locally using `npm run size` in each package. We could configure the repo to report the Size Limit output on each PR listing each package individually.